server/base/src/main/java/org/apache/accumulo/server/tablets/UniqueNameAllocator.java

```diff
@@ -71,31 +70,29 @@ public synchronized String getNextName() {
         throw new RuntimeException(e);
       }
     }
-
     return new String(FastFormat.toZeroPaddedString(next++, 7, Character.MAX_RADIX, new byte[0]),
         UTF_8);
   }
 
   private int getAllocation() {
-    int baseAllocation =
-        context.getConfiguration().getCount(Property.GENERAL_FILENAME_BASE_ALLOCATION);
-    int jitterAllocation =
-        context.getConfiguration().getCount(Property.GENERAL_FILENAME_JITTER_ALLOCATION);
-
-    if (baseAllocation <= 0) {
-      log.warn("{} was set to {}, must be greater than 0. Using the default {}.",
-          Property.GENERAL_FILENAME_BASE_ALLOCATION.getKey(), baseAllocation,
-          DEFAULT_BASE_ALLOCATION);
-      baseAllocation = DEFAULT_BASE_ALLOCATION;
+    int minAllocation = context.getConfiguration().getCount(MIN_PROP);
+    int maxAllocation = context.getConfiguration().getCount(MAX_PROP);
+
+    if (minAllocation <= 0) {
+      int defaultMin = DefaultConfiguration.getInstance().getCount(MIN_PROP);
```

Comment:
Is there a reason to recalculate the default? It could be a static class variable, right?

Reply:
It's not recalculating it, but it is re-parsing the string. I suppose we could save that in a local variable, or memoize it in a memoizing supplier to avoid the string-to-int parse costs, but I don't think it makes much difference.

The main change here is that it's using getCount() instead of a manual parse, which it was doing before.

Reply:
Fixed in 268a573acb14772ca8bd1f03793f40962f0a3ee9